src/entries/popup/pages/messages/SendTransaction/SendTransactionsInfo.tsx

```diff
@@ -461,13 +463,15 @@ function InsuficientGasFunds({
             variant="transparent"
             color="blue"
             onClick={() => {
-              setSelectedToken(nativeAsset);
-              navigate(ROUTES.BRIDGE, { replace: true });
-              onRejectRequest({ preventWindowClose: true });
-              triggerToast({
-                title: i18n.t('approve_request.request_rejected'),
-                description: i18n.t('approve_request.bridge_and_try_again'),
-              });
+              if (nativeAsset) {
+                setSelectedToken(nativeAsset);
```

Comment:
Was there a reason we need this check? The if wrapper here would just mean that the button does nothing when clicked, which would be an awkward state. `hasBridgeableBalance` should be doing the work here to ensure that nativeAsset is not null

Reply:
it was causing a ts error bc nativeAsset could be undefined